app.js

```diff
@@ -1,142 +1,38 @@
 const express = require('express');
+const mongoose = require('mongoose');
 const morgan = require('morgan');
 const cors = require('cors');
 const path = require('path');
-const { v4: uuidv4 } = require('uuid');
-const Joi = require('joi');
-const fs = require('fs/promises');
+require('dotenv').config();
+
+const contactRoutes = require('./routes/api/contacts');
 
 const app = express();
-const PORT = 3000;
+
+const contactsPath = '/Users/molikos/Desktop/GoIT/goit-nodejs-hw-01/goit-nodejs-hw-03/models/contacts.json';
```

Comment:
A co jeżeli ja bedę chciał uruchomić ten server u siebie? Nie mam takiej ścieżki ;) 
Tu trzeba zrobić to w ten sposób
```
constactsPath = __dirname + '/models/contacts.json';
Sprawdź czy to działa. 
Ale od kiedy zaczynamy uzywać bazy danych mongodb to nie potrzebujesz pliku json

Reply:
poprawione